src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 40 * days;
```

Comment:
Consider using a constant variable for the daily rental cost (40) for better readability and maintainability.